client/src/game/level.ts

```
@@ -0,0 +1,150 @@
+          blocked.add(cellKey({ x, z }));
+        }
+      }
+    }
+    reachable = reachableFrom(spawn, config.gridSize, blocked);
+    const openCells = interiorCells - crates.length;
+    if (reachable.size >= openCells * 0.6) break;
+  }
+
+  let terminal = spawn;
+  let bestDist = -1;
+  for (const [key, dist] of Array.from(reachable)) {
```

ESLint

[error] Use spread syntax [...iterable] instead of Array.from(iterable) when no mapper function is provided

(e18e/prefer-spread-syntax)